### PR TITLE
Add Video player page

### LINKED ISSUE #174: Video viewer should allow change of reading speed
Many players allows their users to modify the video reading speed, either more slowly to ease comprehension or faster for speed-viewing (e.g. 0.75x - 1.25x)

### LINKED ISSUE #215: UI Revamp: Single Video Player Page
This is a subtask of #165 

The "Single Video Player" page is where a single video can be viewed by the user. This is where users will be redirected when they click on a video in the "Videos" page (#213). The UI redesign for this page will be based on the following designs:

- Desktop View:

- Mobile View:

### PR DESCRIPTION
**Pull request by dan-niles**
This PR adds the video player for the new UI.
- Set up video.js with support for subtitles and variable playback rates.
- Added `channel_description` and `channel_joined_date` to the `Author` schema so that we can display channel information easily in the video player page.
- Included `webp-hero` for WebP support in unsupported browsers.
- Added a Cypress test for video player page.

`Note:` ogv.js has not been setup yet. I've created #230 to track it seperately.

Closes #215
Closes #174

### CODE REVIEW

**Review by benoit74 — CHANGES_REQUESTED** [review]
Two small inline comments.

Regarding WebP support, I unfortunately cannot test it on my machines. And I'm not even sure I should. I've opened https://github.com/openzim/overview/issues/36 to discuss this topic openly, because I begin to consider we might be loosing our precious time with this WebP / WebM polyfills.

File: zimui/src/views/VideoPlayerView.vue [hunk 1]
```diff
@@ -28,12 +33,70 @@ const fetchData = async function () {
...
+      src: `${video.value?.subtitlePath}/video.${subtitle.code}.vtt`,
+      srclang: subtitle.code,
+      label: subtitle.name
+    }
+  })
+})
+
+const videoOptions = ref({
+  controls: true,
+  autoplay: false,
```
[the remark this answers is not in the document]
Comment (dan-niles) [reply zimui/src/views/VideoPlayerView.vue:58]: Created #233 to track this

File: zimui/src/views/VideoPlayerView.vue [hunk 2]
```diff
@@ -28,12 +33,70 @@ const fetchData = async function () {
...
+  preload: true,
+  fluid: true,
+  responsive: true,
+  enableSmoothSeeking: true,
+  controlBar: { pictureInPictureToggle: false },
+  playbackRates: [0.25, 0.5, 1, 1.5, 2],
```
[the remark this answers is not in the document]
Comment (dan-niles) [reply zimui/src/views/VideoPlayerView.vue:64]: For now, we'll be using `0.25, 0.5, 1, 1.5, 2` according to #174

**Review by benoit74 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**